insights/parsers/pcs_status.py

```diff
@@ -128,10 +129,15 @@ def parse_content(self, content):
                 else:
                     self.data["WARNING"] = [line.strip()]
                 continue
+            if (line.startswith("RemoteNode") or line.startswith("GuestNode") or line.startswith("Node")):
+                linesplit = line.split()
+                bad_node = {}
+                bad_node['name'] = linesplit[1][0:-1]
+                bad_node['status'] = line.split(':')[1][1:]
+                self.bad_nodes.append(bad_node)
```

Comment:
@wduffeeb do you need to add some tests for this?

Reply:
@bfahr test update committed